public/v4/apps/windmill.yml

```diff
@@ -84,12 +81,11 @@ caproverOneClickApp:
     variables:
         - id: $$cap_app_version
           label: Windmill Version
-          defaultValue: '1.99'
+          defaultValue: '1.109.1'
           description: Checkout their github page for the valid tags https://github.com/windmill-labs/windmill/releases
           validRegex: /^([^\s^\/])+$/
         - id: $$cap_wm_base_url
           label: WM Base URL
-          defaultValue: https://$$cap_appname.$$cap_root_domain
```

Comment:
Is this removed intentionally?

Reply:
Yes. I thought that it would be replaced in the setup screen, but as far as I tested, it still appears as the literal string `https://$$cap_appname.$$cap_root_domain`.